### PR TITLE
added HTML

### PR DESCRIPTION
**Pull request by JFCotter**

### CODE REVIEW

File: src/index.htm [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+<html>
+
+<head>
+    <title>My Ageas</title>
+    <link rel="stylesheet" href="/resetstylesheet.css">
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito">
+    <link rel="icon" type="image/x-icon" href="/public/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ghost) [inline src/index.htm]: meta tags should always be on top of the head element

File: src/index.htm [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <title>My Ageas</title>
+    <link rel="stylesheet" href="/resetstylesheet.css">
```
Comment (ghost) [inline src/index.htm]: In this case you can include the rest styles on the top of your styles.css, so you won't have 2 css files.

File: src/index.htm [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+</head>
+
+<body>
+    <header>
+        <a href="https://www.ageas.co.uk/?_ga=2.121409935.647310521.1698137173-364032112.1696943888">
+            <img id="logoWhiteFont" src="/public/logoWhiteFont.svg" alt="Ageas Logo">
+        </a>
+    </header>
+    <div class="flexContainerMain">
+        <main>
```
Comment (ghost) [inline src/index.htm]: the main element needs to be the parent element here, not the div. the order should be:
header, main and footer to be on the same level.

File: src/index.htm [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            <h1 class="intro">Register for an Online Account</h1>
+            <form>
+                <p class="intro">Our easy to use online account lets you manage your policy, view your policy documents
```
Comment (ghost) [inline src/index.htm:23]: you can't have two classes with the same name. Better some sort of convention here e.g. introHeader, introCopy or even better "heading", "intro"

File: src/index.htm [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                <p class="labelDetail">(Please use your email address)</p>
+                <input class="textBox" type="email" id="username" placeholder="example@hotmail.co.uk">
+                <div class="flexContainerInput">
+                    <label for="password">Password</label><img class="helpIcon" src="/public/questionSymbol.png"
+                        alt="Help Icon">
+                </div>
+                <p class="labelDetail">(Between 8 and 16 characters, including at least 1 letter and 1
+                    number and we accept the following special characters!$%^&*#@)</p>
+                <input class="textBox" type="password" id="password">
+                <button class="purpleButton" type="submit">Create my online account</button>
```
Comment (ghost) [inline src/index.htm:69]: best way to apply basic styles to button is if you use a common class (e.g. button) then other classes for button specific styles

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/index.htm (hunk 1 above)
Comment (JFCotter) [reply src/index.htm]: moved to top of head element

File: src/index.htm (hunk 2 above)
Comment (JFCotter) [reply src/index.htm]: done

File: src/index.htm (hunk 3 above)
Comment (JFCotter) [reply src/index.htm]: switched div and main element, main is now the flex container.

File: src/index.htm (hunk 4 above)
Comment (JFCotter) [reply src/index.htm:23]: removed intro class from h1

File: src/index.htm (hunk 5 above)
Comment (JFCotter) [reply src/index.htm:69]: noted, i don't think I will use two classes in this instance, since there is only one style of button in this project

**Pull request merged**